hadoop-hdds/docs/content/security/SecurityAcls.md

```diff
@@ -78,16 +78,86 @@ allows the user to overwrite an existing ozone key.
 6. **Read_ACL** – Allows a user to read the ACL on a specific object.
 7. **Write_ACL** – Allows a user to write the ACL on a specific object.
 
-<h3>Ozone Native ACL APIs</h3>
+There are 2 types of ACLs based on their scope - **Access** and **Default**.<br>
+Access ACLs are limited only to the specific object and not inherited. They control the access to the object itself.
+Only Default ACLs support inheritance. This inheritance is limited to only one level. For example,
```

Comment:
This only one level inheritance is actually not the preferred behavior.  There is an task ongoing to fix it #4738 .  
You can leave this level thing out in this patch,  just mention that DEFAULT ACL will inherit to its descents.